dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -2549,19 +2549,19 @@
       "type": "string",
       "enum": [
         "Unspecified",
-        "MeterPerSecond",
-        "KilometerPerHour",
-        "KilometerPerMinute",
-        "KilometerPerSecond",
-        "MilePerHour",
-        "Knot",
-        "FootPerSecond",
-        "FootPerMinute",
-        "YardPerMinute",
-        "YardPerSecond",
-        "MeterPerMillisecond",
-        "CentimeterPerMillisecond",
-        "KilometerPerMillisecond"
+        "MetersPerSecond",
+        "KilometersPerHour",
+        "KilometersPerMinute",
+        "KilometersPerSecond",
+        "MilesPerHour",
+        "Knots",
+        "FeetPerSecond",
+        "FeetPerMinute",
+        "YardsPerMinute",
+        "YardsPerSecond",
+        "MetersPerMillisecond",
+        "CentimetersPerMillisecond",
+        "KilometersPerMillisecond"
```

Comment:
Just want to make sure that we're okay with diverging from what's defined in CLU. We won't be able to share these definitions between TA/CLU.

https://github.com/Azure/azure-rest-api-specs/blob/756495dd7e0e2f5181039def47a8c85ff0787b66/specification/cognitiveservices/data-plane/Language/stable/2023-04-01/analyzeconversations.json#L980-L998

Reply:
Yes, we are aware we are diverging from the CLU definition.